config/state-machine.php

```diff
@@ -50,6 +50,10 @@
                 'from' => ['printed','dispatched','allocated'],
                 'to' =>  'recorded',
             ],
+            'reject' => [
```

Comment:
Why does reject move to allocated? What does it mean?

Reply:
I chose reject, as the market app is rejecting it from the 'collect'ed vouchers list. The workflow then shunts it back to "allocated".